src/tablespace.c

```diff
@@ -315,47 +309,60 @@ tablespace_attach(PG_FUNCTION_ARGS)
 	tspcname = PG_GETARG_NAME(0);
 	hypertable_oid = PG_GETARG_OID(1);
 
+	tablespace_attach_direct(tspcname, hypertable_oid);
+
+	PG_RETURN_VOID();
+}
+
+void
+tablespace_attach_direct(Name tspcname, Oid hypertable_oid)
```

Comment:
Not a big deal, but does it make more sense to call this `tablespace_attach_internal`?